src/externalized/DefaultContentManager.cc

```diff
@@ -1254,6 +1256,56 @@ void DefaultContentManager::loadVehicles()
 	VehicleModel::validateData(m_vehicles);
 }
 
+void DefaultContentManager::loadTranslationTable()
+{
+#define TRANSLATION_TABLE_SIZE (0x452)
+#define TRANSLATION_FULL_TABLE_SIZE (0x10000)
+
+	m_translationTable.clear();
+	ST::string fullName = "translation_tables/";
+	int tableSize = TRANSLATION_TABLE_SIZE;
+	switch (m_gameVersion)
+	{
+	case GameVersion::DUTCH:
+	case GameVersion::ENGLISH:
+	case GameVersion::GERMAN:
+	case GameVersion::ITALIAN:
+	case GameVersion::POLISH:
+		fullName += "english";
+		break;
+
+	case GameVersion::FRENCH:
+		fullName += "french";
+		break;
+
+	case GameVersion::RUSSIAN:
+		fullName += "russion";
+		break;
+
+	case GameVersion::RUSSIAN_GOLD:
+		fullName += "russion-gold";
+		break;
+
+	case GameVersion::SIMPLIFIED_CHINESE:
+		fullName += "simplifed-chinese";
+		tableSize = TRANSLATION_FULL_TABLE_SIZE;
+		break;
+
+	default:
+		throw std::runtime_error(ST::format("unknown game version {}", static_cast<int>(m_gameVersion)).to_std_string());
+	}
+	fullName += "-translation-table.json";
+
+	m_translationTable.assign(tableSize, 0);
+
+	auto json = readJsonDataFile(fullName);
+	int count = 0;
+	for (auto& element : json->GetArray())
+	{
+		m_translationTable[ST::string(element.GetString()).to_utf16()[0]] = count++;
```

Comment:
Is all this conversion jumping through hoops really needed?

Reply:
Use numbers in json?
```
[
65, /* A */
66, /* B */
67, /* C */
.....
]
```
Json does not support comments. but rapidjson has no complaints about it.

Reply:
That or a map, but I don't know if we need to bother, since I imagine it's still a fast conversion.

Reply:
The original implementation used an array, I think it's fine.
Array is faster than map when running, but the advantage is very small,  can be ignored.